pkg/miscellaneous/miscellaneous.go

```diff
@@ -441,10 +441,6 @@ func GetInitialClusterStateIfScaleup(ctx context.Context, logger logrus.Entry, c
 		return pointer.StringPtr(ClusterStateExisting), nil
 	}
 
-	// fallback - preserves backward compatibility
```

Comment:
removing this condition is fine as it never made much sense anyways to determine a scale up scenario. However in case of a rolling update a new node will be launched, this node should join an existing cluster and should not create a new cluster of its own.